tests/Teams/MessagingExtensionAuth/TeamsAppManifest/manifest.json

```diff
@@ -0,0 +1,73 @@
+{
+  "$schema": "https://developer.microsoft.com/en-us/json-schemas/teams/v1.5/MicrosoftTeams.schema.json",
+  "manifestVersion": "1.5",
+  "version": "1.0",
+  "id": "",
+  "packageName": "com.teams.sample.compose.extension",
```

Comment:
>compose [](start = 35, length = 7)

Feel free to ignore. Not sure if we're going to keep compose as part of description - should this be just messaging?

Reply:
fixed, thanks!